Meta/WPT.sh

```diff
@@ -207,13 +220,26 @@ import_wpt()
         item="${item#https://wpt.live/}"
         INPUT_PATHS[i]="$item"
     done
+
+    TESTS=()
+    mapfile -t TESTS < <( "${ARG0}" list-tests "${INPUT_PATHS[@]}" )
```

Comment:
In #664 we replaced all instances of `mapfile -t` in our shell scripts with a ` while IFS= read -r <thing>` loop. (I'm surprised you missed this @sideshowbarker  :P) .

Anyway, I'm not sure the exact construct to use here, but `mapfile -t` is not compatible with system bash on macOS, which would be nice to keep working as long as possible to not mandate a homebrew-installed shell or some such.

Reply:
> In #664 we replaced all instances of `mapfile -t` in our shell scripts with a ` while IFS= read -r <thing>` loop. (I'm surprised you missed this @sideshowbarker :P) .

I’m surprised too — if you mean it should make the script fail? I mean, when I run it, it doesn’t fail for me, and it writes the correct expectations files, with passes for what should be passing, etc.

So I wonder what I’m doing differently, or wrong (or right…)

Reply:
Probably not using system bash? `/usr/bin/bash WPT.sh list-tests dom`? I'll check it on my MacBook I suppose. I would have presumed that the issue was with mapfile itself, but perhaps it's just a fluke that our use in the CI scripts was buggy in system bash (3.2)

Reply:
> Anyway, I'm not sure the exact construct to use here, but `mapfile -t` is not compatible with system bash on macOS, which would be nice to keep working as long as possible to not mandate a homebrew-installed shell or some such.

ah, I guess that must be it — I’m not using the Apple-installed bash, but instead the homebrew-installed one:

```
$ bash --version
bash --version
GNU bash, version 5.2.37(1)-release (aarch64-apple-darwin24.0.0)
```

I guess I should always test with the Apple one, too

Reply:
I just want to avoid @lukewarlow filing a bug in a few weeks when he tries out the new features of the importer script :D